Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM node:10.7.0-stretch as builder
+WORKDIR /src
+
+COPY client client
+COPY server server
+COPY package.json package.json
+COPY package-lock.json package-lock.json
+COPY webpack.config.prod.js webpack.config.prod.js
+COPY elm-package.json elm-package.json
+RUN npm install && npm run build
+
+FROM registry.opensource.zalan.do/stups/node:8.9.4-alpine-34
+MAINTAINER Sergii Kamenskyi <sergukam@sergukam.com>
+LABEL Description="This Nakadi UI does not check auth by default. Please use Auth plugins for use in production."
+
+WORKDIR /app
+COPY --from=builder /src/dist dist
+COPY --from=builder /src/server server
+COPY --from=builder /src/package.json .
+RUN npm install --only=production
+
+EXPOSE 3000/tcp
+
+ENV HTTP_PORT="3000"
```

Comment:
I prefer to put env var to env file or docker compose file. That way Dockefile looks cleaner.

Reply:
Yes, it is possible. But I want to show to the reader of the Dockerfile what options are possible, it is easier to see here without documentation.